clarification about physical qubit declaration (#202)
Qiskit/openqasm: source/language/types.rst

However, it also means that users or compiler have to explicitly manage
the quantum memory.

.. code-block:: c

   // Declare a qubit
   qubit gamma;
   // Declare a qubit with a Unicode name

>    qubit γ;

Author: ```suggestion
   qubit γ;
   // Declare a qubit register with 20 qubits
   qubit[20] qubit_array;
```

> 

Reviewer: Pls add the qubit register line: `qubit[20] qubit_array;` Seems you've deleted this

Author: added!

However, it also means that users or compiler have to explicitly manage
the quantum memory.

.. code-block:: c

   // Declare a qubit
   qubit gamma;
   // Declare a qubit with a Unicode name
   qubit γ;
   // Declare a qubit register with 20 qubits

>    qubit[20] qubit_array;

Reviewer 2: @blakejohnson Are we allowing this syntax for qubit registers or do we prefer `array[qubit, 20]`?

Reviewer 3: Both are allowed. The types WG will need to discuss whether they are equivalent.

used in lower parts of the compilation stack when emitting physical
circuits.

> The language OpenQASM3 does not impose any limitation on how the discrete space of physical qubits is organized. In other words, it is possible to declared any loose physical qubit:

Reviewer: ```suggestion
The language OpenQASM3 does not impose any limitation on how the discrete space of physical qubits is organized. In other words, it is possible to declare any loose physical qubit:
```

> The language OpenQASM3 does not impose any limitation on how the discrete space of physical qubits is organized. In other words, it is possible to declared any loose physical qubit:

Reviewer: ```suggestion
There is no limitation on how the discrete space of physical qubits is organized. In other words, it is possible to declare any loose physical qubit:
```

.. code-block:: c

   // Declare two physical qubits
   qubit $3;

>    qubit $12;

Reviewer 3: Do physical qubits need to be declared? Seems like I should be able to directly use physical qubit indices without declaring them first...
